### PR TITLE
(WIP) Update traits description

### PR DESCRIPTION
**Pull request by ybw0014**
Some traits' description does not match their effects
need to double-check if I am wrong
Only core traits

### CODE REVIEW

File: src/main/java/io/github/sefiraat/slimetinker/items/componentmaterials/cmfactories/CMCore.java [hunk 1]
```diff
@@ -996,15 +996,15 @@
                                                 CMTraits.PROP_PLATES,
                                                 SupportedPluginsManager.CORE_NOTE,
                                                 "Band",
-                                                "Durability loss + 200%. Damage reduced between 1-25%",
+                                                "Durability loss + 300%. Damage reduced between 1-25%",
```
Comment (Sefiraat) [inline src/main/java/io/github/sefiraat/slimetinker/items/componentmaterials/cmfactories/CMCore.java:999]: For this one the DurabilityEvent#plateBrass method is actualyl the one that is wrong. +200% is intended, I simply screwed up number. Can you undo this and either change the method `friend.setDurabilityMod(friend.getDurabilityMod() + 3);`. Good catch though, you'd have no way to know which way was correct :) !

**Comment by ybw0014** [comment]
The duralium head should reduce damage by 50% and cannot get any tool exp
In `BlockBreakEvents` and `EntityDamageEvents`, the `headDuralium` is defined but not registered, is this a mistake or moved to somewhere else?

**Comment by Sefiraat** [comment]
> The duralium head should reduce damage by 50% and cannot get any tool exp
> In `BlockBreakEvents` and `EntityDamageEvents`, the `headDuralium` is defined but not registered, is this a mistake or moved to somewhere else?

Work was done to move the unbreakable part and seems like I orphaned these methods incorrectly. I have fixed this in the commits I'll be pushing later. The description you have therre is correct - Works when broken, slow +1, 0 Exp and -50% damge

**Comment by ybw0014** [comment]
This PR should be marked as WIP because I may find more.

**Comment by ybw0014** [comment]
For iron links and iron singularity links, both reduce damage by 10%, but iron +10% player exp while iron sing +20%.
The in-game desc says +10% durability loss instead of reducing damage. 

---

The iron's event is registered in `TickEvents`.
The iron sing's event is named `linksIron`.

Are these two functions in the wrong places?

**Comment by Sefiraat** [comment]
> For iron links and iron singularity links, both reduce damage by 10%, but iron +10% player exp while iron sing +20%.
> The in-game desc says +10% durability loss instead of reducing damage.
> 
> The iron's event is registered in `TickEvents`.
> The iron sing's event is named `linksIron`.
> 
> Are these two functions in the wrong places?

`EntityDamageEvent#linksIron` needs to be renamed `EntityDamageEvent#linksSignIron`
And then `linksIron` should be in `EntityDamage`, whoops.

**Comment by ybw0014** [comment]
> > For iron links and iron singularity links, both reduce damage by 10%, but iron +10% player exp while iron sing +20%.
> > The in-game desc says +10% durability loss instead of reducing damage.
> > The iron's event is registered in `TickEvents`.
> > The iron sing's event is named `linksIron`.
> > Are these two functions in the wrong places?
> 
> `EntityDamageEvent#linksIron` needs to be renamed `EntityDamageEvent#linksSignIron`
> And then `linksIron` should be in `EntityDamage`, whoops.

So it should be reducing damage?

**Comment by ybw0014** [comment]
i am bad in git management, reopen a PR later

**Pull request closed without merging**